modules/sdk-coin-apt/src/apt.ts

```diff
@@ -79,18 +79,14 @@ export class Apt extends BaseCoin {
     return true;
   }
 
-  getTransaction(coinConfig: Readonly<StaticsBaseCoin>): Transaction {
-    return new TransferTransaction(coinConfig);
-  }
-
   async verifyTransaction(params: VerifyTransactionOptions): Promise<boolean> {
     const coinConfig = coins.get(this.getChain());
     const { txPrebuild: txPrebuild, txParams: txParams } = params;
-    const transaction = this.getTransaction(coinConfig);
     const rawTx = txPrebuild.txHex;
     if (!rawTx) {
       throw new Error('missing required tx prebuild property txHex');
     }
+    const transaction = utils.getTransactionFromRawTx(rawTx, coinConfig);
```

Comment:
nit: let's make this function a static function of Transaction class and rename it to `fromRawTx()`.
```suggestion
    const transaction = Transaction.fromRawTx(rawTx, coinConfig);
```